hw/top_earlgrey/data/ip/chip_otp_ctrl_testplan.hjson

```diff
@@ -0,0 +1,208 @@
+
+            X-ref'ed with chip_sw_lc_ctrl_broadcast test, which verifies the connectivity of the LC
+            decoded outputs to other IPs.
+            '''
+      features: ["OTP_CTRL.PARTITION.SECRET2"]
+      stage: V2
+      si_stage: SV1
+      tests: [
+        // lc_dft_en_i
+        "chip_prim_tl_access",
+        // lc_check_byp_en_i
+        "chip_sw_lc_ctrl_transition",
+        // lc_seed_hw_rd_en_i, lc_creator_seed_sw_rw_en_i, also checks lc_keymgr_en_i since it uses
+        // the keymgr.
+        "chip_sw_otp_ctrl_lc_signals_test_unlocked0",
+        "chip_sw_otp_ctrl_lc_signals_dev",
+        "chip_sw_otp_ctrl_lc_signals_prod",
+        "chip_sw_otp_ctrl_lc_signals_rma",
+        ]
+    }
+    {
+      name: chip_sw_otp_prim_tl_access
+      desc: '''Verify that the SW can read / write the prim tlul interface.
+
+            - The prim tlul interface is a open source placeholder for the closed source CSRs that
+              will be implemented in a translation 'shim'.
+            - Verify that when `lc_dft_en_i` is On, this region can be read / written to by the SW.
+              When `lc_dft_en_i` is Off, accessing this region will result in a TLUL error.
+            '''
+      stage: V2
+      si_stage: None
+      tests: ["chip_prim_tl_access"]
+    }
+    {
+      name: chip_sw_otp_ctrl_vendor_test_csr_access
+      desc: '''
+            Verify the vendor test control access in raw, test_*, dev, prod, and rma LC states.
+
+            - Boot the chip successively in raw, test_*, dev, prod and rma LC states.
+            - Verify that the SW is able to access the vendor test control and status registers in
+              raw, test_* and rma LC states.
+              In open source environment, this check is implemented by probing the OTP_CTRL's
+              `lc_otp_vendor_test_i` port.
+            - Verify that in dev / prod LC states, the vendor status always reads back 0s regardless
+              of what is programmed into the vendor test control register.
+            '''
+      features: ["OTP_CTRL.VENDOR_TEST_PARTITION"]
+      stage: V3
+      si_stage: SV1
+      tests: ["chip_sw_otp_ctrl_vendor_test_csr_access"]
+    }
+    {
+      name: chip_sw_otp_ctrl_escalation
+      desc: '''Verify escalation from otp_ctrl macro fatal error.
+
+            - Inject ECC fatal error into OTP macro's HW cfg partition, and read back this macro
+              via DAI interface.
+            - Because this fatal error will immediately turn off CPU, so the DV sequence will probe
+              the alert interface to make sure alert and escalation is triggered.
+
+            X'ref with chip_sw_all_escalation_resets.
+            '''
+      features: ["OTP_CTRL.ERROR_HANDLING"]
+      stage: V3
+      si_stage: SV3
+      tests: ["chip_sw_otp_ctrl_escalation"]
+    }
+    {
+      name: otp_ctrl_calibration
+      desc: "Verify all calibrated parameters are programmed during manufacturing stage calibration test phase."
```

Comment:
Some of this is tested by side-effect, like the clock calibration can be checked via clkmgr clock counts, and there are tests for that.